tests/0124-openssl_invalid_engine.c

```diff
@@ -35,22 +35,21 @@ int main_0124_openssl_invalid_engine (int argc, char **argv) {
         rd_kafka_conf_res_t res;
 
         test_conf_init(&conf, NULL, 30);
-        if (rd_kafka_conf_set(conf, "security.protocol", "ssl", 
-                              errstr, sizeof(errstr)) != RD_KAFKA_CONF_OK)
-                TEST_FAIL("%s", errstr);
-
         res = rd_kafka_conf_set(conf, "ssl.engine.location", "invalid_path", 
                                 errstr, sizeof(errstr));
 
         if (res == RD_KAFKA_CONF_UNKNOWN) {
-                rd_kafka_destroy(conf);
```

Comment:
You need to destroy the conf or it will leak.

Reply:
sorry didn't realize there is rd_kafka_conf_destroy(conf);